source/unix/uv_unix_fs.c

```diff
@@ -77,9 +77,11 @@
 
 #define PATH                                                                  \
   do {                                                                        \
-    (req)->path = strdup(path);                                               \
-    if ((req)->path == NULL)                                                  \
-      return -ENOMEM;                                                         \
+    if (path) {                                                               \
```

Comment:
IMHO, we should follow the current style and explicitly write the NULL in the condition.
```c
if (path != NULL) {
```